src/main/scala/TransactionsScoringApp.scala

```diff
@@ -1,2 +1,6 @@
 object TransactionsScoringApp extends App {
+  while(true) {
+    println("Hello")
+    Thread.sleep(5000)
+  }
 }
```

Comment:
We usually use packages like in Java-world, e.g. `com.github.ntsaturov.transactions.scoring`